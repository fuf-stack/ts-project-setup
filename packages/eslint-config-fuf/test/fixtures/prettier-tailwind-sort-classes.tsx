/* eslint-disable */

/* eslint-disable import-x/no-extraneous-dependencies */
/* eslint-disable import-x/no-unresolved */

import React from 'react';

import { cn, tv } from '@fuf-stack/pixel-utils';

// tailwind variants
const _variants = tv({
  base: 'px-4 bg-blue-500 text-base rounded text-white py-2'
});

const MyComponent = () => {
  const _outsideCn = cn('px-4 bg-blue-500 text-base rounded text-white py-2');

  return (
    <div className="text-white rounded">
      <div className={cn('text-white rounded')}>with cn</div>
    </div>
  );
};

export default MyComponent;
