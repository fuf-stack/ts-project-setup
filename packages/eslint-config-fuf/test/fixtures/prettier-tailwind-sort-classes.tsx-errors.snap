[
  {
    "message": "Replace `px-4·bg-blue-500·text-base·rounded·text-white·py-2'` with `rounded·bg-blue-500·px-4·py-2·text-base·text-white',`",
    "ruleId": "prettier/prettier",
  },
  {
    "message": "Replace `px-4·bg-blue-500·text-base·rounded·text-white·py-2` with `rounded·bg-blue-500·px-4·py-2·text-base·text-white`",
    "ruleId": "prettier/prettier",
  },
  {
    "message": "Replace `text-white·rounded` with `rounded·text-white`",
    "ruleId": "prettier/prettier",
  },
  {
    "message": "Replace `text-white·rounded` with `rounded·text-white`",
    "ruleId": "prettier/prettier",
  },
]